install_files/ansible-base/roles/build-ossec-deb-pkg/library/ossec_urls.py

```diff
@@ -96,23 +90,23 @@ def parse_checksums(self):
 
 
 def main():
-    module = AnsibleModule(
+    module = AnsibleModule( # noqa E405
```

Comment:
s/E405/F405/

Reply:
This one was doubly bad because it was ignoring _all_ flake8 violations, not merely `E405` as it appears. Instead of `# noqa F405`, it should be `# noqa: F405` to permit just that one violation.